unittests/src/main/java/uk/ac/manchester/tornado/unittests/common/TornadoTestBase.java

```diff
@@ -71,6 +74,10 @@ public void before() {
         }
     }
 
+    private boolean getVirtualDeviceEnabled() {
+        return Boolean.parseBoolean(System.getProperty("tornado.virtual.device", "False"));
```

Comment:
Move this flag to `TornadoOptions`

Reply:
We already have this in the `TornadoOptions`, but it is in the `runtime` module. 
The `unittests` module only has as a dependency the `tornado-api` module. Are we fine with adding `tornado-runtime` as a dependency to the `unittests` module ?

Reply:
M... no. We will break the licenses. Let's keep it here.